scripts/package-info/devnet.json

```
@@ -1,21 +1,21 @@
 {
-	"adminAddress": "0xdf2f0a373e65c48b0c4ac636e03183a573bb36af88216511bbede6704f877b6e",
-	"adminCap": "0x7aef1ed31a153f20c70b4d4f5f935902128467bc3fd56bc7bfcfde854603933c",
-	"auctionPackageId": "0xbd7e5f5f0272aa93472ebd6dabcff16433db3816bb54c0fdcda30c450f181900",
-	"coins": {
-		"IOTA": {
-			"type": "0x0000000000000000000000000000000000000000000000000000000000000002::iota::IOTA",
-			"metadataId": "0xf7ceb0424ca93f77858c74aebda8bcd5ffe0f32b82ebcc38afb3ba643d03a5f8"
-		}
-	},
-	"denyListPackageId": "0x3525137488ce0534f83956079314b79f68b8439176b207a3af78faa67c7032d3",
-	"iotaNames": "0xdad5289ef0d64f8f3b4d72522907f3f67109fa00bfbcba2dd03c68084f1dfc89",
-	"packageId": "0xe27899d691184f66821f8fed5e7c26f3c65b26921947956435a655c8d7efc573",
-	"packageIdPricing": "0xe27899d691184f66821f8fed5e7c26f3c65b26921947956435a655c8d7efc573",
-	"paymentsPackageId": "0x8e1d3fafb70764eccc2e6b61812daf0a4db40db3c5cea515bf4d390f11016030",
-	"publisherId": "0x205afb5bb6bb68790e99a99d2628cc5020ca7655f063edcca7a366de510536d1",
-	"registryTableId": "0xff608b2b0d500b4d0cb25ff165bc3e01fce9bf3ef7fb002840b814d304a08b2a",
-	"reverseRegistryTableId": "0x1c2eddd6c4f7510b35a9de575d9ccb1ad640de6aa3a5626937c21c9c62beaeed",
-	"subNamesPackageId": "0x45573a8859b9e26e73a42b9c31cec9a41080d1c9b61738ee89c2d58f7c11ece6",
-	"tempSubdomainsProxyPackageId": "0xf43e05a098dd8a339d478907418f42b30eddf661b029a48f313edee1420e22fe"
-}
+  "adminAddress": "0xb98bc58cd2cae4c621b47ed0a3bbed2e1dcb78da60d578625384dad3a231c6fb",
+  "adminCap": "0x84096eeb81f666bc8e7b14803b387617ad92aa796b5f84ddc752a513b273fc34",
+  "auctionPackageId": "0x5e7a300e640f645a4030aeb507c7be16909e6fa9711e7ca2d4397bbd967d5c50",
+  "coins": {
+    "IOTA": {
+      "type": "0x0000000000000000000000000000000000000000000000000000000000000002::iota::IOTA",
+      "metadataId": "0xf7ceb0424ca93f77858c74aebda8bcd5ffe0f32b82ebcc38afb3ba643d03a5f8"
+    }
+  },
+  "denyListPackageId": "0xe2d9b88ccfd3c752152873385e3da7d2d849e0bd98e91db68e7bd16e41fe8345",
+  "iotaNames": "0x54a8a67fad7aa279429e08824e03481dd8b268779353d299d7f8edaa8b8c13b7",
+  "packageId": "0x3ec4826f1d6e0d9f00680b2e9a7a41f03788ee610b3d11c24f41ab0ae71da39f",
+  "packageIdPricing": "0x3ec4826f1d6e0d9f00680b2e9a7a41f03788ee610b3d11c24f41ab0ae71da39f",
+  "paymentsPackageId": "0x882f88d252a650649f490e96e32e53979758fdda645863ca856d83c72d5e0e72",
+  "publisherId": "0xc10d3ab816bdbbb0b04c45b4fafebaad9d739fcd57bfc805120193bc7eeee25f",
+  "registryTableId": "0xef24c78e8c085e29760d37b287fc16647f0f578e8d22f18dd65f655285afad3e",
+  "reverseRegistryTableId": "0x566dc13eafceaf8c3487ee2c41464553839ef4d50937c63741e359c98080c7b6",
+  "subNamesPackageId": "0x2e541b250f53d45e9b4cb866be2ab3d8815015a249a094e63b196cc184278925",
+  "tempSubdomainsProxyPackageId": "0x3f2927e8e78a094bb98271fd73e06a3aa056edfecbf32dddb8a65d30de9b8a3f"
+}
```

Probably complains about this
```suggestion
}

```

---

Should we check this through dprint instead of pnpm lint?

---

Here pnpm lint complains, not sure if dpring would modify it

---

Would you want to ignore it from pnpm lint?

---

prettier doesn't even see it locally, weird

---

It does actually, I messed up